### ISSUE TITLE
run kdoctor -v fail with error:   ✖ Xcode requires to perform the First Launch tasks

### ISSUE THREAD

**Issue by powerzd** [Kotlin/kdoctor#34]
I tried to run kdoctor on my terminal, but it always fail. Here is the log: 

[✓] Operation System
[✓] Java
[✓] Android Studio
[✖] Xcode
  ✖ Xcode requires to perform the First Launch tasks
    Launch Xcode and complete setup
[✖] Cocoapods
  ✖ cocoapods not found
    Get cocoapods from https://guides.cocoapods.org/using/getting-started.html#installation

Conclusion:
  ✖ KDoctor has diagnosed one or more problems while checking your environment.
    Please check the output for problem description and possible solutions.
My Xcode version is 14.2, it was launched successfully and run a project perfectly. What should i do to fix the problem.

**Comment by WillWidlof** [comment]
Worth a look, but if you previously installed the XCode Command Line Tools. Then you might need to manually switch your Xcode location to look for the full app and not the Comand Line Tools with:

`sudo xcode-select -s /Applications/Xcode.app/Contents/Developer`

**Comment by powerzd** [comment]
cool, it works, thanks

**Comment by eldaroid** [comment]
THNX!

**Comment by terrakok** [comment]
Fixed in "1.0.1"

```
kdoctor
Environment diagnose (to see all details, use -v option):
[✓] Operation System
[✓] Java
[✓] Android Studio
[✖] Xcode
  ✖ Current command line tools: /Library/Developer/CommandLineTools
    You have to select command line tools bundled to Xcode
    Command line tools can be configured in Xcode -> Settings -> Locations -> Locations
[✓] Cocoapods

Conclusion:
  ✖ KDoctor has diagnosed one or more problems while checking your environment.
    Please check the output for problem description and possible solutions.
```
